WEB/Src/DependencyCollector/DependencyCollector/TelemetryDiagnosticSourceListener.cs

```diff
@@ -50,8 +50,23 @@ public void OnEvent(KeyValuePair<string, object> evnt, DiagnosticListener diagno
             }
 
             // properly fill dependency telemetry operation context
-            telemetry.Context.Operation.Id = currentActivity.RootId;
-            telemetry.Context.Operation.ParentId = currentActivity.ParentId;
+            if (currentActivity.IdFormat == ActivityIdFormat.W3C)
```

Comment:
Could you add unit test for this? There should already be unit tests for this class,

Reply:
I'll do it, thanks for the review